$MirOS: ports/www/lynx/patches/patch-lynx_help_lynx_help_main_html,v 1.6 2011/06/21 17:35:53 tg Exp $

	we install these, so link to them
	also add ddg to search engines

--- lynx_help/lynx_help_main.html.orig	Mon Feb 13 02:45:53 2012
+++ lynx_help/lynx_help_main.html	Sat Mar  3 18:20:22 2012
@@ -33,6 +33,12 @@
 
     <li><a href="lynx-dev.html">About Lynx-Dev</a> &mdash; the
     developers &amp; how to contact them</li>
+
+    <li><a href="../lynx_doc/samples/">Lynx samples</a> &mdash; the
+    samples contained in the source distribution</li>
+
+    <li><a href="../lynx_doc/test/">Lynx tests</a> &mdash; the
+    test cases contained in the source distribution</li>
   </ul>
 
   <h2>Other sources of Lynx help:</h2>
@@ -142,6 +148,8 @@
 
     <li><a href="http://www.ask.com/">Ask.com</a></li>
 
+    <li><a href="https://duckduckgo.com/lite/">Duck Duck Go</a></li>
+
     <li><a href="http://www.excite.com/">Excite</a></li>
 
     <li><a href="http://www.go.com/">GO.com</a></li>
@@ -175,5 +183,7 @@
 
     <li><a href="http://mail.yahoo.com/">Yahoo! Mail</a></li>
   </ul>
+<hr />
+<pre>$MirOS: src/gnu/usr.bin/lynx/lynx_help/lynx_help_main.html,v 1.8 2012/02/19 19:38:01 tg Exp $</pre>
 </body>
 </html>
